### ISSUE TITLE
Fix is not being applied to grouped-imports

### ISSUE THREAD

**Issue by po5i** [palantir/tslint#4453]
### Bug Report

- __TSLint version__: 5.12.0
- __TypeScript version__: 3.0.3
- __Running TSLint via__: Node.js API

#### TypeScript code being linted

```ts
import * as R from "ramda";

import { Request, Response } from "express";
import { ValidationError } from "sequelize";
```

with `tslint.json` configuration:

```json
"ordered-imports": [
      true,
      {
        "import-sources-order": "lowercase-first",
        "grouped-imports": true,
        "named-imports-order": "lowercase-first"
      }
    ],
```

#### Actual behavior

`ERROR: myfile.ts:3:1 - Import sources of the same type (package, same folder, different folder) must be grouped together.`

#### Expected behavior

Message saying the file was fixed since this rule "has fixer".

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is deprecated** and no longer accepting pull requests other than security fixes. See #4534. ☠️ 
We recommend you instead use **[typescript-eslint](https://typescript-eslint.io)** to lint your TypeScript code with **ESLint**. ✅

👋 It was a pleasure open sourcing with you!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
